prep-javascript-objects/main.js

```diff
@@ -0,0 +1,21 @@
+const person = {
+  firstName: 'Eduardo',
+  lastName: 'Gallegos',
+  job: 'Electrician',
+  previousJob: 'Door Dasher',
```

Comment:
make sure you add these properties to the object after the initial creation using dot and bracket notation